pkgs/applications/networking/cluster/mesos/default.nix

```diff
@@ -193,7 +192,7 @@ in stdenv.mkDerivation rec {
     mkdir -p $out/share/java
     cp src/java/target/mesos-*.jar $out/share/java
 
-    MESOS_NATIVE_JAVA_LIBRARY=$out/lib/libmesos.${soext}
+    MESOS_NATIVE_JAVA_LIBRARY=$out/lib/libmesos${stdenv.hostPlatform.extensions.sharedLibrary}
```

Comment:
pfft, I prefered `.soext` 😝

Reply:
I only inlined of another long line was a worse offender 😝